concourse/pipelines/rhui-release.yaml

```diff
@@ -54,7 +54,27 @@ jobs:
         - --region=us-west1
         - --project=google.com:rhel-infra
         path: gcloud
-    task: wait-rhua
+    task: rhua-wait-for-version-target
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - compute
+        - instance-groups
+        - managed
+        - wait-until
+        - --stable
```

Comment:
this is in the wait-for-version-target step?

Reply:
the jsonnet renderer puts the `task` key at the bottom of the object (I'm revisiting the jsonnet now to replace what's here)